log/logger.go

```diff
@@ -26,8 +26,12 @@ func ConfigureLogger(ctx *cli.Context) {
 	logLevel := logrus.Level(ctx.GlobalInt("verbosity"))
 	Log.SetLevel(logLevel)
 
+	var timestampFormat string
 	if logLevel >= logrus.DebugLevel {
 		Log.SetReportCaller(true)
+		timestampFormat = "01-02|15:04:05.999999999"
```

Comment:
two points:
A) we only want 3 digit precision
B) we want milliseconds on all print levels